### PR TITLE
Update ups access point delivery article

### PR DESCRIPTION
**Pull request by MaWeXi**
With the new interface we need to update the article for UPS PUDO shipments and delete the article for ups access point notifications as it is no longer supported.

**remove the following lines:**

- "you have to provide the exact address of the UPS access point using the drop_off_point attribute"
- "you also have to use the additional service ups_access_point_notification [see example]"

**add the following lines to the remaining lines:**

- "the recipients address has to be provided as you normally would using the to attribute"
- "to.first_name is mandatory"
- "to.last_name is mandatory"
- "to.email is mandatory"
- "drop_off_point.drop_off_point_id is mandatory and has to be 9 characters long (can be determined through our [pudo provider](https://developers.shipcloud.io/swagger-ui/#/default/get_pickup_dropoff_locations))"
- "drop_off_point.type has to be parcel_shop"

---

[sc-30429](https://app.shortcut.com/shipcloud/story/30429)

### CODE REVIEW

File: _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html [hunk 1]
```diff
@@ -17,19 +17,24 @@ <h4 id="ship-webservice---access-point-delivery" class="text-blue">
...
-    you also have to use the additional service
-    <code><em>ups_access_point_notification</em></code>
-    (<a data-controller="toggle-element"
-      data-target="#ship_webservice_access_point_delivery_togglebox" aria-expanded="false"
-      aria-controls="collapseExample" class="cursor-pointer">see example</a>)
+    <code><em>drop_off_point.drop_off_point_id</em></code> is manadatory and has to be 9 characters long (can be determined through our
+    <a style="white-space: nowrap;" href="{{ site.baseurl }}/swagger-ui/#/default/get_pickup_dropoff_locations">
+      pudo provider
+    </a>
+    )
```
Comment (MaWeXi) [inline _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html:33]: This produces an unwanted whitespace which I do not know how to fix, like this:
(can be determined through our [pudo provicer](https://developers.shipcloud.io/swagger-ui/#/default/get_pickup_dropoff_locations) )

Comment (SteffenHam) [reply _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html:33]: Could you put everything in one line? Would that fix that?
```
<a style="white-space: nowrap;" href="{{ site.baseurl }}/swagger-ui/#/default/get_pickup_dropoff_locations">pudo provider</a>)

Comment (MaWeXi) [reply _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html]: It did fix the problem, thank you.

File: _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html [hunk 2]
```diff
@@ -46,50 +48,38 @@ <h4 id="ship-webservice---access-point-delivery" class="text-blue">
...
-    "city": "Hamburg",
-    "country": "DE"
+      "first_name": "Serge",
+      "last_name": "Sender",
+      "company": "Sender Corp.",
+      "street": "Sender Str.",
+      "street_no": "99",
+      "zip_code": "20148",
+      "city": "Hamburg",
+      "country": "DE"
```
Comment (SteffenHam) [inline _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html:58]: All JSON lines are now indented with two tabs or 4 spaces. Is that necessary?

Comment (MaWeXi) [reply _includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html]: Fixed in [76d9947](https://github.com/shipcloud/shipcloud-developer-portal/pull/296/commits/76d9947974f94c077cd13758207d89839618596d)

**Review by SteffenHam — APPROVED** [review]
RTM

**Pull request merged**